### PR TITLE
Site Blank Page Template

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Committed site blank page template

### CODE REVIEW

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/hclecomm-empty-page/.content.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:mix="http://www.jcp.org/jcr/mix/1.0"
+    jcr:mixinTypes="[mix:lockable]"
+    jcr:primaryType="cq:Template"
+    ranking="{Long}1">
+    <jcr:content
+        jcr:description="Generic template for web pages"
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/hclecomm-empty-page/.content.xml]: @DeepaliAgrwl  update description

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/hclecomm-empty-page/.content.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        jcr:primaryType="cq:PageContent"
+        jcr:title="custom hclecomm Empty Page"/>
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/hclecomm-empty-page/.content.xml]: @DeepaliAgrwl  update jcr:title

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/.content.xml [hunk 3]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:rep="internal"
+    jcr:mixinTypes="[rep:AccessControllable]"
     jcr:primaryType="cq:Page">
     <empty-experience-fragment/>
-    <empty-page/>
     <rep:policy/>
+    <hclecomm-blank-page/>
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/.content.xml:7]: @DeepaliAgrwl  : why do we have this here ?

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/hclecomm-empty-page/.content.xml (hunk 2 above)
Comment (DeepaliAgrwl) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/hclecomm-empty-page/.content.xml]: done

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/.content.xml (hunk 3 above)
Comment (DeepaliAgrwl) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/.content.xml:7]: @rRishabh this file contents the list of all the templates inside this folder. It is automatically updated when we create or delete any template inside this folder .

**Pull request merged**